lib/spack/spack/test/spec_syntax.py

```diff
@@ -882,6 +896,59 @@ def test_cli_spec_roundtrip(args, expected):
     assert expected == output_string
 
 
+@pytest.mark.parametrize(
+    "spec_str,toolchain,expected_roundtrip",
```

Comment:
minor: I think we should start expanding these:

```suggestion
    ["spec_str", "toolchain", "expected_roundtrip"],
```

Reply:
Let's leave this for later - no time atm for style issues like this one :slightly_smiling_face: